default/templates/default/index.html

```diff
@@ -81,4 +81,70 @@
         </div>
     </a>
 </div>
+<div class="panel panel-default info">
+<div class="panel-body descriptive">
+    <p>
+        {% blocktrans trimmed %}
+        <a href="https://standard.open-contracting.org/" target="_blank">OCDS</a>
+        Toucan offers a set of transformation tools.<br>
+        You sometimes need to transform OCDS data before you can use it or publish it.
+        {% endblocktrans %}
+    </p>
+    <p>
+        {% trans "Toucan allows you to get OCDS data ready for use, in your browser:" %}
+    </p>
+    <p>
+        <ul>
+            {% blocktrans trimmed %}
+            <li> Create a
+            <a href="https://standard.open-contracting.org/latest/en/schema/release_package/"
+               target="_blank">release package</a> from individual
+            <a href="https://standard.open-contracting.org/latest/en/getting_started/releases_and_records/#releases"
+               target="_blank">releases</a><br></li>
+            {% endblocktrans %}
+
+            {% blocktrans trimmed %}
+            <li> Combine many small packages into a large package <br></li>
```

Comment:
You can use the `{% trans "" %}` tag for short lines like this one. Also there is no need to use `<br>` tags here.